packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -43,7 +40,44 @@ extension ClassPrivateViewExtension on Class {
 /// An extension on [InboundReferences] which allows for access to
 /// VM internal fields.
 extension InboundReferenceExtension on InboundReferences {
+  static const referencesKey = 'references';
+  static const parentWordOffsetKey = '_parentWordOffset';
+
   int? parentWordOffset(int inboundReferenceIndex) {
     return json![referencesKey]?[inboundReferenceIndex]?[parentWordOffsetKey];
   }
 }
+
+class HeapStats {
```

Comment:
Do you happen to know why this is appearing as your change? I added this class in my PR that landed last week.

Reply:
Hmm.. there was a conflict when I merged so I added the closing curly bracket for the InboundReferences extension. The heapStats still appears as your change to me...